readme.md

```diff
@@ -519,6 +519,28 @@ const Demo = (
 );
 ```
 
+The `<Text>` compoment is a simple wrapper around [the `chalk` API](https://github.com/chalk/chalk#api) it supports all of the listed
+chalk methods as `props`.
+
+```jsx
+import { Text } from "ink"
+
+<Text
```

Comment:
This isn't really a functioning example. I'd provide several examples, that demonstrate each way of setting color: `red`, `bgRed`, `rgb={[255, 255, 255]}`, etc.